### ISSUE TITLE
[VS] Single project - quickly switching targets and F5 causes error

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#16516]
### Current behavior

When I switch targets and F5 in quick succession, I get intermittent error in console:

```
1>Error: Cannot access a disposed object.
1>Object name: 'Microsoft.VisualStudio.ProjectSystem.ConfiguredProjectImpl'.
```

Second F5 works fine

### Expected behavior

No error

### Workaround

- Run the debug again **or**
- Wait a bit longer between the switches between targets

### Works on UWP/WinUI

None

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### IDE

Visual Studio 2022

**Comment by MartinZikmund** [comment]
I guess this is must be already tracked somewhere, if yes, feel free to close as dupe

**Comment by jeromelaban** [comment]
This is caused by:
1. [net8.0 must be first for WSL debugging to work](https://developercommunity.visualstudio.com/t/net80-desktop-must-be-first-for-WSL-deb/10643714)
2. [net8.0 must be first for WebAssembly publishing/debugging to work
](https://developercommunity.visualstudio.com/t/net80-must-be-first-for-WebAssembly-pub/10643720)
3. [net8.0-windows10 needs to be first for WinAppSDK Hot reload to work](https://developercommunity.visualstudio.com/t/net80-windows10-needs-to-be-first-for-W/10643724)

**Comment by ADD-David-Antolin** [comment]
@jeromelaban notice they (bot) closed you the second one as duplicated of the first one.

**Comment by jeromelaban** [comment]
@ADD-David-Antolin yes we noticed, and it's known on the Microsoft side, it will be adjusted.

**Comment by MartinZikmund** [comment]
This is no longer happening
